### PR TITLE
feat(Graphic, redux)

### PR DESCRIPTION
**Pull request by MaxiV95**
image

When selecting a button it lights up

image

### CODE REVIEW

File: src/redux/store.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { createStore, applyMiddleware, compose } from "redux";
+import thunkMiddleware from "redux-thunk";
+import reducer from "./reducer";
+
+//conectamos nuestra app con la extension
+const composeEnhancer = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```
Comment (joser098) [inline src/redux/store.js]: If we use redux toolkit? it does not need all this configuration.

Comment (MaxiV95) [reply src/redux/store.js]: Good suggestion! Thank you!

**Comment by MaxiV95** [comment]
@joser098 Do you think it is a good selection of names?

File: src/components/graphic/Graphic.jsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+					>
+						{period.toString() + "%"}
+					</button>
+				))}
+			</span>
+
+			<br />
+			{/* graph here */}
+
+			<h3>ESPECTRO DE PELIGRO UNIFORME + ESPECTRO DE PELIGRO</h3>
```
Comment (manduinca) [inline src/components/graphic/Graphic.jsx]: Maybe: "Espectro de Peligro Uniforme y Espectro de Diseño"

File: src/components/graphic/Graphic.jsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+			<br />
+
+			<button className={style.buttonToMap}>
+				Latitude: {location.lat}, Longitude: {location.lng}
+			</button>
+
+			<br />
+
+			<h3>PROBABILIDAD ANUAL DE EXCEDENCIA</h3>
```
Comment (manduinca) [inline src/components/graphic/Graphic.jsx]: I think is better: "Probabilidad Anual de Excedencia"

**Review by manduinca — CHANGES_REQUESTED** [review]
Please all communication with users in spanish.

**Comment by MaxiV95** [comment]
I just modularized, because by adding functionality it's going to grow a lot

**Comment by MaxiV95** [comment]
Push the rest of the code so they can review the structure.
Some small changes are missing from the last commit

File: src/components/graphic/Graphic.jsx (hunk 2 above)
Comment (MaxiV95) [reply src/components/graphic/Graphic.jsx]: Good choice

File: src/components/graphic/Graphic.jsx (hunk 3 above)
Comment (MaxiV95) [reply src/components/graphic/Graphic.jsx]: Good choice

**Review by FacundoPucheta — DISMISSED** [review]
Seems good

**Comment by MaxiV95** [comment]
Combined selectors
Independent buttons

Ready for final review!

image
image

**Review by manduinca — APPROVED** [review]
LGTM!

**Review by SebaSatelier — APPROVED** [review]

**Pull request merged**